RetrospectiveExtension.Frontend/webpack.config.js

```diff
@@ -49,13 +50,14 @@ module.exports = {
   plugins: [
     new MomentLocalesPlugin(),
     new ESLintPlugin(),
+    new Dotenv(),
     new webpack.ProvidePlugin({
       process: 'process/browser',
     }),
     new webpack.DefinePlugin({
       'process.env': {
-        BUILD_BUILDNUMBER: JSON.stringify(process.env.BUILD_BUILDNUMBER),
-        'process.env.NODE_ENV' : JSON.stringify(process.env.NODE_ENV || 'production'),
+        BUILD_BUILDNUMBER: JSON.stringify(process.env.BUILD_BUILDNUMBER)
+        // 'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV || 'production'),
```

Comment:
Why did you comment out this line?

Reply:
will uncomment it, was testing some changes with env variables